charts/k8ssandra/Chart.yaml

```diff
@@ -1,6 +1,7 @@
 apiVersion: v2
 name: k8ssandra
 description: |
+  DEPRECATED please see https://github.com/k8ssandra/k8ssandra-operator
   Provisions and configures an instance of the entire K8ssandra stack. This includes Apache Cassandra, Stargate, Reaper, Medusa, Prometheus, and Grafana.
```

Comment:
Might want to reword the description slightly since k8ssandra-operator does not provision Prometheus or Grafana.

Reply:
That's for the k8ssandra-operator chart right? Not the k8ssandra chart?

Reply:
Yup, I got my wires crossed. Sorry about that.